docs/Load-Sample-Cancer-Study.md

```diff
@@ -1,14 +1,24 @@
 # Loading a Sample Study
 
-Once you have initialized MySQL with the seed database, you are ready to import a sample cancer study.  This is recommended, so that you can verify everything is working.
+Once you have initialized MySQL with the seed database, you are ready to import a sample cancer study. This is recommended to verify that everything is working correctly.
 
-## Sample Study
+The cBioPortal distribution includes a [small dummy study, `study_es_0`](https://github.com/cBioPortal/cbioportal/tree/master/core/src/test/scripts/test_data/study_es_0), which contains all datatypes supported by cBioPortal. This document describes how to import the prerequisites for the sample study and how to import the study itself.
 
-The cBioPortal distribution includes a [small dummy study, study_es_0](https://github.com/cBioPortal/cbioportal/tree/master/core/src/test/scripts/test_data/study_es_0), which you can use to verify that everything is working properly.
+#### Import Gene Panels and Gene Sets for Sample Study
 
-## Validating the Sample Study
+In 2017 two new datatypes have been added which both require an additional action before loading study data. Gene panels and gene sets have to be imported before respectively gene panel study data and gene set scoring data can be added to the database.
```

Comment:
Is it important to tell the people who'd use this documentation the story of when it was introduced? I'm afraid it might only complicate the message.